services/src/main/java/edu/unc/lib/dl/cdr/services/rest/modify/AddContainerController.java

```diff
@@ -51,39 +51,39 @@
     @Autowired
     private AddContainerService addContainerService;
 
-    @RequestMapping(value = "edit/create/adminUnit/{id}", method = RequestMethod.POST)
+    @RequestMapping(value = "edit/create/adminUnit/{id}/{label}", method = RequestMethod.POST)
```

Comment:
A label can have a lot of weird stuff in it, like /'s. I feel like I'd expect the label to be passed along as either part of the body of the request or a header, and we aren't generally using headers like this in the CDR. So perhaps `@RequestParam("label")`? I see some of our other POST methods like the mark for deletion stuff are using that.

Reply:
makes sense. done!